.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+name: Release
+on:
+  push:
+    tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: coursier/setup-action@v1
+        with:
+          jvm: zulu:8
+      - run: sbt -Dsun.net.client.defaultReadTimeout=60000 -Dsun.net.client.defaultConnectTimeout=60000 -Dspark.version=3.5.3 -v ci-release
```

Comment:
Does it use the tag as a version? Or what would be the version of the published artifact?

Reply:
This one will use tag as the version. for example:
git tag v1.0.1 -m 'v1.0.1'
git push origin v1.0.1
will trigger the pipeline for version 1.0.1